packages/sp1-ics07-tendermint-utils/src/rpc.rs

```diff
@@ -153,18 +154,19 @@ impl TendermintRpcExt for HttpClient {
             .ok_or_else(|| anyhow::anyhow!("No consensus state found"))
     }
 
-    async fn prove_path(&self, path: &[Vec<u8>], height: u32) -> Result<(Vec<u8>, MerkleProof)> {
+    async fn prove_path(&self, path: &[Vec<u8>], height: u64) -> Result<(Vec<u8>, MerkleProof)> {
+        let previous_height = (height - 1).try_into()?;
         let res = self
             .abci_query(
                 Some(format!("store/{}/key", std::str::from_utf8(&path[0])?)),
                 path[1..].concat(),
                 // Proof height should be the block before the target block.
-                Some((height - 1).into()),
+                Some(previous_height),
                 true,
             )
             .await?;
 
-        if u32::try_from(res.height.value())? + 1 != height {
+        if res.height.value() + 1 != height {
```

Comment:
we still use inline +1 here. So, we should continue using inline +1 above for consistency